gemrb/unhardcoded/iwd/gemrb.ini

```diff
@@ -143,6 +143,7 @@ ReverseDoor = 0
 ReverseToHit = 1
 SaveForHalfDamage = 0
 SelectiveMagicRes = 0
+SellableCritsNoConv = 1
```

Comment:
You need to add these also to the other gemrb.ini files, just with 0.

Reply:
Thanks, Jaka, I've pushed the files I missed.

Reply:
There's one more, for the demo, just elsewhere in the tree.

Reply:
Fixed and pushed.

Did I miss anything else?

Reply:
Wait, I screwed that one up, sorry.

Reply:
The stuff in my first two comments is still not addressed, yeah.

Reply:
I'll sort it out.  Still tracing through the code.

Reply:
Should I make another game flag to enable the selling of items which originally have no bits set for the destructible flag?

Reply:
Why, is behaviour different between the games?

Reply:
I can't find where it's being set, but the "critical" items appear to also have the "destructible" bit set to false, the destructible flag doesn't exist in the original iwd/bg1.

If I add a condition for a !"destructible" flag after the condition for critical/not conversable it won't permit the sale.  If I don't add the condition it will permit the sale of indestructible items.

Reply:
It's in Interface::SanitizeItem. But again, until the behaviour is proven to be different among games, a game flag makes no sense. There's already a check in this function, I'm just asking what the proper order of the two is, so we can match the originals.

Reply:
Thanks for the leg up, Jaka. I'll go read and break some more code.

Reply:
btw, I agree about the gameflag, that was just frustration talking, sorry.

I have a single test case working with modified interface.cpp code in IWD, I'll unpack bg2 and pst over the weekend and try them out.  

It's been too long since I played pst, too bad it's just for bughunting this time, but it'll be nice to see it again. The GemRB code runs really well, even runs smoothly on a crappy OLPC notebook I use.

Reply:
No need to test pst stores, as they're not really different and hard to find. It's also almost pointless to test with gemrb, the main questions can only be answered by the originals.

Reply:
Gorion's  Scroll (scrl3z) has the critical flag and can be sold to Bentley Mirrorshade in the original (non-EE) bg+totsc.

I'll continue testing...